### ISSUE TITLE
Toggle Switch field type is not accessible by keyboard navigation

### ISSUE THREAD

**Issue by kevinwhoffman** [CalderaWP/Caldera-Forms#1811]
## What Version Of Caldera Forms, WordPress and PHP Are You Using?

WordPress Version: 4.8
PHP Version: 7.0.16
MySQL Version: 5.6.36
Caldera Forms Version: 1.5.3.1

## Does Your Issue Persist When You Disable All Other Plugins and Switch To The Default Theme?

Yes

## What Is The Unexpected Behaviour?

Toggle Switch field type uses `a` elements without the `href` attribute defined. As a result, these buttons are not accessible by keyboard navigation. The browser skips over them when tabbing through the DOM.

## Recommended Solution

The Toggle Switch field should use `button` elements since `a` is reserved for hyperlinks, and these buttons are not linking anywhere. `button` is the semantically correct element and will also solve the issue of tab focus with keyboard navigation.

**Comment by mathetos** [comment]
Would love to see some traction on this. Currently a blind user wouldn't be able to submit a support form to us at all because we use the toggles as the primary conditionals for our support form.

**Comment by Shelob9** [comment]
This should not be that hard as the toggle switch is just controlling a
hidden radio field. We should be able to use aria attributes to communicate
that the toggle labels the radio.
